src/SFA.DAS.Apim.Developer.Web/Startup.cs

```diff
@@ -111,26 +112,32 @@ public void ConfigureServices(IServiceCollection services)
                      services.AddAuthenticationCookie(serviceParameters.AuthenticationType);
                 }
 
-                services.AddMaMenuConfiguration(RouteNames.EmployerSignOut, clientId,_configuration["Environment"]);
+                services.AddMaMenuConfiguration(RouteNames.EmployerSignOut, clientId, _configuration["Environment"]);
                 services.Configure<ExternalLinksConfiguration>(_configuration.GetSection(ExternalLinksConfiguration.ApimDeveloperExternalLinksConfiguration));
                 services.AddSingleton(new ProviderSharedUIConfiguration());
             }
             else if (serviceParameters.AuthenticationType == AuthenticationType.Provider)
             {
                 services.AddProviderUiServiceRegistration(_configuration);
                 services.AddProviderAuthenticationServices();
-                if (_configuration["StubAuth"] != null && _configuration["StubAuth"]
-                        .Equals("true", StringComparison.CurrentCultureIgnoreCase))
+
+                if (_configuration["StubAuth"] != null && _configuration["StubAuth"].Equals("true", StringComparison.CurrentCultureIgnoreCase))
                 {
                     services.AddProviderStubAuthentication();
                 }
                 else
                 {
-                    services.AddAndConfigureProviderAuthentication(_configuration
-                        .GetSection(nameof(ProviderIdams))
-                        .Get<ProviderIdams>());    
+                    if (_configuration["ApimDeveloperWeb:UseDfESignIn"] != null && _configuration["ApimDeveloperWeb:UseDfESignIn"].Equals("true", StringComparison.CurrentCultureIgnoreCase))
+                    {
+                        services.AddAndConfigureDfESignInAuthentication(_configuration, $"{typeof(AddServiceRegistrationExtension).Assembly.GetName().Name}.Auth", typeof(ProviderAccountPostAuthenticationClaimsHandler));
```

Comment:
Dont we want to pass null as the custom claims here? Otherwise it will throw a `NotImplementedException` as the implementation isn't done in `ProviderAccountPostAuthenticationClaimsHandler`

Reply:
Thanks, agreed, removed